storagenode/operator.go

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package storagenode
+
+import (
+	"fmt"
+	"regexp"
+
+	"go.uber.org/zap"
+)
+
+// OperatorConfig defines properties related to storage node operator metadata
+type OperatorConfig struct {
+	Email  string `user:"true" help:"operator email address" default:""`
+	Wallet string `user:"true" help:"operator wallet address" default:""`
+}
+
+// Verify verifies whether operator config is valid.
+func (c OperatorConfig) Verify(log *zap.Logger) error {
+	if err := isOperatorEmailValid(log, c.Email); err != nil {
+		return err
+	}
+	if err := isOperatorWalletValid(log, c.Wallet); err != nil {
+		return err
+	}
+	return nil
+}
+
+func isOperatorEmailValid(log *zap.Logger, email string) error {
+	if email == "" {
+		log.Sugar().Warn("Operator email address isn't specified.")
```

Comment:
We shouldn't need to use a sugared logger in this file.

Reply:
looks like that's the way it was originally implemented
https://github.com/storj/storj/blob/master/pkg/kademlia/config.go#L74